manual-install/latest.yml

```diff
@@ -110,7 +110,6 @@ services:
       - ADMIN_USER=admin
       - ADMIN_PASSWORD=${NEXTCLOUD_PASSWORD}
       - NEXTCLOUD_DATA_DIR=/mnt/ncdata
-      - OVERWRITEHOST=${NC_DOMAIN}
```

Comment:
I guess we will need to get rid of this then... or not? I would honestly like to keep it if possible...

Reply:
@st3iny do you know maybe?

Reply:
I think we can keep it. The domain will not be rewritten in redirects if it is trusted.

Reply:
Nvm, I just tested this locally and the domain is rewritten on redirects (e.g. to default app). So we need to remove `overwritehost` etc. which could be problematic in case of some reverse proxies.

Reply:
In theory they should not use a reverse proxy because caddy is generating the cert so I hope we will be fine...

Reply:
Yeah but Caddy *is* the rproxy. You need to make sure to set the correct headers (FORWARDED) etc. I think we can test this sufficiently though.

Reply:
I think caddy does this automatically fortunately :)